itc4j-core/src/test/java/itc4j/StampTest.java

```diff
@@ -118,7 +118,9 @@ public void testLeq() {
         Stamp s1 = new Stamp();
         Stamp s2 = new Stamp();
         Assert.assertTrue(s1.leq(s2.event()));
+        Assert.assertTrue(Causality.lessThanEquals(s1, s2.event()));
         Assert.assertFalse(s2.event().leq(s1));
+        Assert.assertFalse(Causality.lessThanEquals(s2.event(), s1));
```

Comment:
This class is only supposed to test the `Stamp` class. Can you create a `CausalityTest` class to test the `Causality` class and move this into that test class?
Also, tests for the other methods in the `Causality` class would be awesome.